include/hardware.h

```diff
@@ -70,6 +70,9 @@ void CAPTURE_VideoStop();
 // Gravis UltraSound configuration and initialization
 void GUS_AddConfigSection(const config_ptr_t &conf);
 
+// IBM Music Feature Card configuration and initialization
+void IMF_AddConfigSection(const config_ptr_t& conf);
```

Comment:
I think it's abbreviated as `IMFC` or `MFC`. I'd prefer `IMFC`
https://en.wikipedia.org/wiki/IBM_Music_Feature_Card